os/kernel/binary_manager/binary_manager.c

```diff
@@ -130,6 +153,11 @@ int binary_manager(int argc, char *argv[])
 	attr.mq_msgsize = sizeof(binmgr_request_t);
 	attr.mq_flags = 0;
 
+	if (BIN_STATE(KERNEL_IDX) != BINARY_RUNNING || g_bin_count <= 0) {
+		dbg("Partition is not set for kernel and user binary! PANIC!! \n");
```

Comment:
How about changing dbg to bmdbg?

Reply:
This is intended code for showing wrong partition settings in configuration.